### PR TITLE
Fix typos in audio register descriptions

### PR DESCRIPTION
**Pull request by QuinnPainter**

### CODE REVIEW

File: hardware.inc [hunk 1]
```diff
@@ -384,7 +384,7 @@ DEF AUDVOL_VIN_RIGHT EQU %00001000 ; SO1
 ; -- Bit 3   - Output sound 4 to SO1 terminal
 ; -- Bit 2   - Output sound 3 to SO1 terminal
 ; -- Bit 1   - Output sound 2 to SO1 terminal
-; -- Bit 0   - Output sound 0 to SO1 terminal
+; -- Bit 0   - Output sound 1 to SO1 terminal
```
Comment (ISSOtm) [inline hardware.inc:387]: Arguably, "sound" should be changed to "channel". Also, maybe we could finally document which of SO1 and SO2 is left/right? ><"

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: hardware.inc (hunk 1 above)
Comment (QuinnPainter) [reply hardware.inc]: Yes, I agree. Just added these new changes to the pull request.

**Review by ISSOtm — APPROVED** [review]

File: hardware.inc [hunk 2]
```diff
@@ -361,10 +361,10 @@ DEF rAUD4GO EQU rNR44
 ; -- AUDVOL/NR50 ($FF24)
 ; -- Channel control / ON-OFF / Volume (R/W)
 ; --
-; -- Bit 7   - Vin->SO2 ON/OFF (Vin??)
-; -- Bit 6-4 - SO2 output level (volume) (# 0-7)
-; -- Bit 3   - Vin->SO1 ON/OFF (Vin??)
-; -- Bit 2-0 - SO1 output level (volume) (# 0-7)
+; -- Bit 7   - Vin->SO2 ON/OFF (left)
+; -- Bit 6-4 - SO2 output level (left volume) (# 0-7)
```
Comment (avivace) [inline hardware.inc:365]:
```suggestion
; -- Bit 6-4 - SO2 output level (left channel) (# 0-7)

```

File: hardware.inc [hunk 3]
```diff
@@ -361,10 +361,10 @@ DEF rAUD4GO EQU rNR44
...
 ; -- Channel control / ON-OFF / Volume (R/W)
 ; --
-; -- Bit 7   - Vin->SO2 ON/OFF (Vin??)
-; -- Bit 6-4 - SO2 output level (volume) (# 0-7)
-; -- Bit 3   - Vin->SO1 ON/OFF (Vin??)
-; -- Bit 2-0 - SO1 output level (volume) (# 0-7)
+; -- Bit 7   - Vin->SO2 ON/OFF (left)
+; -- Bit 6-4 - SO2 output level (left volume) (# 0-7)
+; -- Bit 3   - Vin->SO1 ON/OFF (right)
+; -- Bit 2-0 - SO1 output level (right volume) (# 0-7)
```
Comment (avivace) [inline hardware.inc:367]:
```suggestion
; -- Bit 2-0 - SO1 output level (right channel) (# 0-7)

```

**Review by avivace — CHANGES_REQUESTED** [review]

File: hardware.inc (hunk 2 above)
Comment (ISSOtm) [reply hardware.inc:365]: This is confusing with the four *input* "channels", so "speaker", "terminal", or "output" is better; though the latter is redundant with the left part... do we need the parens at all?

Comment (avivace) [reply hardware.inc:365]: I agree with speaker and output

**Review by avivace — APPROVED** [review]

**Pull request merged**